### ISSUE TITLE
Pull: Unable to redirect output from inside a bash script

### ISSUE THREAD

**Issue by atrauzzi** [ansible/ansible#10820]
I have a bash script that is kicking off `ansible-pull`:

```
/usr/local/bin/ansible-pull --inventory-file="localhost," --directory=/var/local/sync --url=$GIT_REPOSITORY --key-file=~/.ssh/id_rsa --accept-host-key --only-if-changed &>> /var/log/sync
```

If I manually trigger the pull, everything works.  But when I review `/var/log/sync`, I end up with the following:

```
    sys.exit(main(sys.argv[1:]))
  File "/usr/local/bin/ansible-pull", line 207, in main
    rc, out, err = cmd_functions.run_cmd(cmd, live=True)
  File "/usr/local/lib/python2.7/dist-packages/ansible/utils/cmd_functions.py", line 31, in run_cmd
    p = subprocess.Popen(cmdargs, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
  File "/usr/lib/python2.7/subprocess.py", line 679, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1259, in _execute_child
    raise child_exception
OSError: [Errno 2] No such file or directory
```

This also happens if I attempt to redirect output from my cron job.  This also happens when I use the `2>&1` redirect as well.

**Comment by sivel** [comment]
I believe this issue is the same described in the fix at https://github.com/ansible/ansible/pull/10487

**Comment by atrauzzi** [comment]
Nice, thanks for digging that up!

**Comment by bcoca** [comment]
closing via already merged #10487
